src/layer/AbstractSentinelHubV1OrV2Layer.ts

```diff
@@ -76,11 +76,11 @@ export class AbstractSentinelHubV1OrV2Layer extends AbstractLayer {
       crs: { type: 'name', properties: { name: bbox.crs.urn } },
       coordinates: [
         [
-          [bbox.minY, bbox.maxX],
-          [bbox.maxY, bbox.maxX],
-          [bbox.maxY, bbox.minX],
-          [bbox.minY, bbox.minX],
-          [bbox.minY, bbox.maxX],
+          [bbox.minX, bbox.minY],
+          [bbox.maxX, bbox.minY],
+          [bbox.maxX, bbox.maxY],
+          [bbox.minX, bbox.maxY],
+          [bbox.minX, bbox.minY],
         ],
       ],
```

Comment:
Is there any info about this somewhere to check?

Reply:
This SO answer seems to summarize it: https://stackoverflow.com/a/57653830 - or the linked [GeoJSON specs](https://tools.ietf.org/html/rfc7946#section-3.1.1):

> The first two elements are longitude and latitude, or
   easting and northing, precisely in that order and using decimal
   numbers.

That said, we are inconsistent in that we use x and y instead of longitude/latitude or easting/northing... Still, I think this fix makes it _less wrong_, but we should revisit how we handle CRSs and BBoxes. I would however prefer to leave this for the future if you agree?

Reply:
Thanks!
Yeah, of course we can revisit this in the future. I just wanted to know what it is referred to, so the links are more than enough.